mangaki/mangaki/management/commands/compare.py

```diff
@@ -17,6 +17,7 @@
 FILENAMES = {
     'movies': 'ratings-ml.csv',
     'mangas': 'ratings.csv',
+    'dummy': '../mangaki/tests/data/ratings.csv',
```

Comment:
ahem
paths, paths, paths

Seriously: add a `FIXME`, we should change how we handle our paths in this command.

Reply:
Done.